logic/labrynna.go

```diff
@@ -133,7 +137,8 @@ var labrynnaNodes = map[string]*Node{
 	"defeat great moblin": AndSlot("ridge west present", "pegasus satchel",
 		"bracelet"),
 	"ridge upper present": Or(And("ridge base present", "switch hook"),
-		And("defeat great moblin", "feather")),
+		And("defeat great moblin", "feather"),
+		And("ridge base past west", "switch hook", "currents")),
```

Comment:
`And("ridge base present", "switch hook")` covers the case of `And("ridge base past west", "switch hook", "currents")`.

Reply:
Ah, that it does. Should this be removed then? Would it be ok to leave a comment saying as much?

Reply:
I think it should just be removed; I think documenting logic redundancies is \*generally* a net loss in terms of readability.